### DISCUSSION TITLE
Global CSS cannot be imported from files other than your Custom <App> error when creating new app with npx create-next-app@latest on windows 10

### DISCUSSION THREAD

**Discussion by gabster-code** [vercel/next.js#47295] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

    Operating System:
      Platform: win32
      Arch: x64
      Version: Windows 10 Pro
    Binaries:
      Node: 16.17.0
      npm: N/A
      Yarn: N/A
      pnpm: N/A
    Relevant packages:
      next: 12.2.6-canary.8
      eslint-config-next: 12.2.5
      react: 18.2.0
      react-dom: 18.2.0

### How are you deploying your application? (if relevant)

locally using npx create-next-app@latest

### Describe the Bug

This is a newly generated app using  `npx create-next-app@latest` command and using `npm run dev` to check if the boilerplate is running correctly. 

![image](https://user-images.githubusercontent.com/88597385/188277665-a0e4d485-a615-4030-ae50-1198861eef79.png)

### Expected Behavior

Expected behavior is a running NextJS Boilerplate site that signifies the created app is working correctly

### Link to reproduction

https://github.com/gabster-code/100daysofcode/tree/main/Activity%20Progress/Day%204_5/WhitelistDapp/hardhat-tutorial/my-app

### To Reproduce

`npx create-next-app@latest`
`npm run dev`

UPDATE: 
I found a template I pushed to Vercel earlier(which is ironically just the base template) and cloned my repo for that.
SAME ERROR OCCURED

Link to vercel site:
https://vercel.com/gabster-code/nextjs/GfxUABTnF2S9hK9XYXsfzwx4cApg

Link to repo I cloned: 
https://github.com/gabster-code/nextjs/tree/main

![image](https://user-images.githubusercontent.com/88597385/188277716-86062779-0cdc-4ae4-8a48-74b1cab3a63f.png)

**Comment by gabster-code** [comment]
Anyone who can help please?

**Comment by grayaustinc** [comment]
I just tried running it and was able to run it just fine with no errors.  Here are the steps I made when running the repository.
![0](https://user-images.githubusercontent.com/57951789/188334239-c8b49495-b316-4271-8799-d3c9b14af106.png)

1. Make sure that you are in the directory `100daysofcode-main\100daysofcode-main\Activity Progress\Day 6\gabsterNFT\my-app`  when running commands for the app. (you can use [cd](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/cd) or start vs code and open using the `my-app` folder)
2. run `npm install`
3. run `npm run dev`

I can only assume based on the screenshots that you were trying to run NextJs outside of the project since the repo that you provided holds multiple projects and not just one.  I am basing that on the fact that there are a lot of `../` in the error message.

Also, all the links that you provided show a 404 not found message.  If they work for you, it may have something to do with them being private.  I used the repo that you cloned, so the steps above should work for you.

Another thing, the repo has already been initialized so you should not have to run `npx create-next-app@latest` at all.  create-next-app is only supposed to be used for creating a boilerplate project and not for a project that already exists.

**Comment by johanroug** [comment]
you can only use a global CSS file from within your _app.js file. If you want to use component specific files, you need to use CSS modules. e.g. "my-file.module.css"
